### ISSUE TITLE
DeFiWallet problem

### ISSUE THREAD

**Issue by TacoTuesdayy** [DeFiCh/app#1060]
premise: I create localClient = new Client (new HttpProvider (`http: // $ {user}: $ {password} @ $ {address}: $ {port} /`), options).

this.localClient.account.getTokenBalances (). then ((data) => {...}
when this code is executed (or also .blockchain and .poolpair) this error is displayed: 
(node: 66113) UnhandledPromiseRejectionWarning: SyntaxError: Unknown symbol "Work" (char 0)

**Comment by thedoublejay** [comment]
Hi @TacoTuesdayy, are you coding this locally or you're having some issue on your app?

**Comment by TacoTuesdayy** [comment]
I'm coding it locally by connecting to port 8555 of the dfiwallet

**Comment by thedoublejay** [comment]
@TacoTuesdayy Do you have node also locally? Did you run this step before running it locally - https://github.com/DeFiCh/app#setup-the-required-binary

**Comment by TacoTuesdayy** [comment]
I don't have a node, I ran a daemon on Linux

**Comment by izzycsy** [comment]
Hi @TacoTuesdayy , update [v2.7.0](https://github.com/DeFiCh/app/releases/tag/v2.7.0) is out, this is a **mandatory update**.  Please read through the release notes before updating. _This comment is to notify you about the update._

**Comment by TacoTuesdayy** [comment]
I upgraded but the problem persists

**Comment by izzycsy** [comment]
Hi @TacoTuesdayy , update [v2.7.1](https://github.com/DeFiCh/app/releases/tag/v2.7.1) is out, this is a **mandatory update**. Kindly `backup wallet` before updating.  Please read through the release notes before updating. _This comment is to notify you about the update._

**Comment by thedoublejay** [comment]
@TacoTuesdayy I'm not familiar with your current setup. Where are your nodes hosted?

**Comment by TacoTuesdayy** [comment]
I started the defiwallet(AppImage) on Linux and locally I connect to port 8555

**Comment by thedoublejay** [comment]
Yes @TacoTuesdayy but you still need to have a running node. Can you verify if you have that?
